hmac_authentication/tests/test_hmac_authentication.py

```diff
@@ -129,45 +111,47 @@ def setUp(self):
             'PATH_INFO': '/foo/bar?baz=quux%2Fxyzzy#plugh',
         }
 
-    def test_validate_request_no_signature(self):
-        result, header, computed = self.auth.validate_request(self.environ)
-        self.assertEqual(HmacAuth.NO_SIGNATURE, result)
+    def test_authenticate_request_no_signature(self):
+        result, header, computed = self.auth.authenticate_request(self.environ)
+        self.assertEqual(AuthenticationResultCodes.NO_SIGNATURE, result)
```

Comment:
One (of many) reasons that I prefer py.test over nose / unittest is for more readable assertions that don't require developers to learn a new API. Using py.test, this becomes `assert result == AuthenticationResultCodes.NO_SIGNATURE`, the next line `assert header is None`, etc.